modules/apps/frontend-taglib/frontend-taglib-clay/src/main/resources/META-INF/resources/dataset_display/management_bar/components/filters/index.js

```diff
@@ -0,0 +1,68 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+import ClayLoadingIndicator from '@clayui/loading-indicator';
+import React, {useEffect, useState} from 'react';
+
+import {getComponentByModuleUrl} from '../../../utilities/modules';
+import AutocompleteFilter from './AutocompleteFilter';
+import CheckboxesFilter from './CheckboxesFilter';
+import DateRangeFilter from './DateRangeFilter';
+import NumberFilter from './NumberFilter';
+import RadioFilter from './RadioFilter';
+import TextFilter from './TextFilter';
+
+export const filterIdToComponentMap = {
+	autocomplete: AutocompleteFilter,
+	checkbox: CheckboxesFilter,
+	dateRange: DateRangeFilter,
+	number: NumberFilter,
+	radio: RadioFilter,
+	text: TextFilter,
```

Comment:
![leaning-tower](https://user-images.githubusercontent.com/7074/87536012-adb6fa00-c698-11ea-9bcf-875a5eed2e27.jpg)

I love it when code has a nice shape. 😂